### DISCUSSION TITLE
ArgoCD how to install CRDs from Helm in App of Apps

### DISCUSSION THREAD

**Discussion by moritz31** [argoproj/argo-cd#14008] [Q&A]
Hey guys, 

we use the app of apps concept and install the kube-prometheus-stack crds as an own helm resource in an applicationset (we also use sync waves there and set the crd applications to the lowest value so that they run first).
We now have in the same applicationset two prometheus rules that should get created. 
If we now create a new cluster the applicationset fails to sync because of this two resources. I already found the following in the documentation https://argo-cd.readthedocs.io/en/stable/user-guide/sync-options/#skip-dry-run-for-new-custom-resources-types but with some tinkering arround this would mean that we need to disable dry run for the full applicationset. Shouldn't it be with sync waves, that argo automatically knows based on the sync waves that this should work ?

**Comment by zivlazar0v** [comment]
any news on that?

**Comment by vasylherman** [comment]
also interested
